src/pages/CreateKas/partials/DialogUsers.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useState } from "react";
+import { BaseDialog, DialogContent } from "@components/Dialog";
+import { UserType } from "@services/CreateKas";
+import useGetUser from "../hooks/useGetUser";
+import glassmorphism from "@utils/glassmorphism";
+import { Search } from "@mui/icons-material";
+import { DialogTitle } from "@mui/material";
+
+interface Props {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+const DialogUsers: React.FC<Props> = ({ isOpen, onClose }) => {
+  const { users, loading, fetchUsers, selectUser } = useGetUser();
+  const [searchTerm, setSearchTerm] = useState("");
+  const [filteredUsers, setFilteredUsers] = useState<UserType[]>([]);
+  const [activeNpm, setActiveNpm] = useState<string | null>(null);
+
+  useEffect(() => {
+    if (isOpen) {
+      fetchUsers();
+      const storedUser = localStorage.getItem("selectedUser");
+      if (storedUser) {
+        const user = JSON.parse(storedUser);
+        setActiveNpm(user.npm);
+      }
+    }
+  }, [isOpen]);
+
+  useEffect(() => {
+    setFilteredUsers(
+      users.filter(
+        (user) =>
+          user.npm.toLowerCase().includes(searchTerm.toLowerCase()) ||
+          user.name.toLowerCase().includes(searchTerm.toLowerCase()),
+      ),
+    );
+  }, [searchTerm, users]);
+
+  const handleSearchChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchTerm(e.target.value);
+  };
+
+  const handleUserSelect = (user: UserType) => {
+    selectUser(user);
+    setActiveNpm(user.npm);
+    localStorage.setItem("selectedUser", JSON.stringify(user));
+    onClose();
+  };
+
+  return (
+    <BaseDialog open={isOpen} onClose={onClose}>
+      <DialogTitle
+        fontSize="0.9rem"
+        sx={{
+          padding: "0.8rem 1.2rem",
+          backgroundColor: "#323232",
+          borderBottom: "1px solid #55555590",
+        }}
+      >
+        <div>Search Your NPM</div>
+        {filteredUsers.length === 0 ? (
+          <div className="italic text-neutral-300">
+            Note: If npm is not found, please register your account to admin
+          </div>
+        ) : (
+          ""
+        )}
```

Comment:
```suggestion
        {filteredUsers.length === 0 && (
          <div className="italic text-neutral-300">
            Note: If npm is not found, please register your account to admin
          </div>
        )}
```
harusnya kayak gini bisa, karena elsenya cuma empty string